xcconfigs/Base.xcconfig

```diff
@@ -4,7 +4,7 @@
 DD_SWIFT_SDK_PRODUCT_NAME=DatadogCore
 DD_CR_SDK_PRODUCT_NAME=DatadogCrashReporting
 
-ARCHS[sdk=iphoneos*]=$(ARCHS_STANDARD) arm64e
+ARCHS[sdk=iphoneos*][config=Release]=$(ARCHS_STANDARD) arm64e
```

Comment:
`KSCrash` is added as an SPM dependency in Xcode. But it seems that Xcode compiles the SPM dependency for `arm64` only in Debug mode when targeting generic destination such as `-destination "generic/platform=iOS"`. There is no need to build for `arm64e` in Debug and having it breaks the SPI doc build.